spread.yaml

```diff
@@ -17,11 +17,16 @@ backends:
         systems:
             - ubuntu-16.04-64-grub
```

Comment:
Can we please do the same with those, having ubuntu-16.04-64 (no -grub suffix) and have the "image:" right below it. I'll rename the images themselves to not have the -grub suffix later, but this will already make things more consistent across core and non-core images, and fix the problem raised below.